scripts/k8s-storage/volumesnapshotclass-cephfs.yaml.in

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshotClass
+metadata:
+  name: k8s-storage-e2e-cephfs
+driver: cephfs.csi.ceph.com
+parameters:
+  clusterID: @@CLUSTER_ID@@
+  csi.storage.k8s.io/snapshotter-secret-name: csi-cephfs-secret
+  csi.storage.k8s.io/snapshotter-secret-namespace: default
```

Comment:
should the namespace not be `rook-ceph`?

Reply:
we use rook created secrets, i was also confused but after thinking a lot and checking other places we are good 👍🏻

Reply:
Can you explain why `sc-rbd.yaml.in` uses `rook-ceph` for all namespaces?

```yaml
csi.storage.k8s.io/provisioner-secret-namespace: rook-ceph
```

Reply:
The `csi-cephfs-secret` is also not created by Rook, I guess?

Reply:
mistake, i though we are talking about why we are using `rook-ceph` which i was added initially but messed up somehow, updated to use rook created secret

Reply:
@nixpanic updated now PTAL